validator/beacon/service.go

```diff
@@ -33,13 +33,13 @@ type Config struct {
 	ProposerChanBuf int
 }
 
-// DefaultConfig options for the beacon client service.
+// DefaultConfig options for the beacon validator service.
 func DefaultConfig() *Config {
 	return &Config{AttesterChanBuf: 5, ProposerChanBuf: 5}
 }
 
-// NewBeaconClient instantiates a service that interacts with a beacon node.
-func NewBeaconClient(ctx context.Context, cfg *Config, rpcClient types.RPCClient) *Service {
+// NewBeaconValidator instantiates a service that interacts with a beacon node.
+func NewBeaconValidator(ctx context.Context, cfg *Config, rpcClient types.RPCClient) *Service {
```

Comment:
I think this should still be `NewBeaconClient`, since this is a wrapper for the beacon's rpc client.